index.html

```diff
@@ -4,10 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/styles.css">
   <title>Luis Zubia First Porfolio</title>
 </head>
 <body>
 
+  <header class="header">
+    <p>Luis Zubia</p>
+    <nav>
+      <a href="#">
+        <img class="icon" src="/imgs/menu-icon.png" alt="Menu icon">
```

Comment:
- [ ] The icon link is not working kindly, remove the `/` at the begining of the link.